mast/processor.py

```diff
@@ -83,17 +86,16 @@ def __calculate_orthodromic_distance(self, coords: list) -> float:
             buffer += self.EARTH_RADIUS * (2 * asin(sqrt(pom)))
         return buffer
 
-    def landing_cleanup(self):
+    def landing_cleanup(self, input_file: str):
         """
         Purpose of this method is clean up landing zone.
         """
         try:
-            for input_file in os.listdir(self.LANDING_DIR):
-                os.remove(os.path.join(self.LANDING_DIR, input_file))
-                logging.info(f"File {input_file} has been removed successfully.")
+            os.remove(os.path.join(self.LANDING_DIR, input_file))
+            logging.info(f"File {input_file} has been removed successfully.")
         except Exception as ex:
             logging.warning("Deletion was unsuccessful!", ex)
 
 
 if __name__ == '__main__':
-    print(list(GPXProcessor().process_input_data()))
+    print(list(GPXProcessor().process_input_data('test.gpx')))
```

Comment:
Please avoid hard-wiring file names. You can still specify the file name in command line arg - sys.argv[1]